Datadog/Example/Scenarios/RUM/RUMScenarios.swift

```diff
@@ -74,6 +74,38 @@ final class RUMModalViewsAutoInstrumentationScenario: TestScenario {
     private class Predicate: UIKitRUMViewsPredicate {
         func rumView(for viewController: UIViewController) -> RUMView? {
             if let viewName = viewController.accessibilityLabel {
+                print(viewName)
+                return .init(name: viewName)
+            } else {
+                return nil
+            }
+        }
+    }
+
+    func configureSDK(builder: Datadog.Configuration.Builder) {
```

Comment:
I don't think it's used 🤔

Reply:
It is. It's called after the scenario boots. For some reason you can't set a breakpoint in it though.